# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/src/xpcprivate.h
# Commit: 942a92874526
# Full Hash: 942a92874526e7a18167af89e358d69cbaeab268
# Author: Boris Zbarsky <bzbarsky@mit.edu>
# Date: 2020-02-22 09:52:08
# Regressor Bug: 267645
# File Overlap Count: 1
# Description:
#   Bug 267645 part 1.  Add a way to ask for DOMException to be exposed in sandboxes. r=bholley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63278
# ==============================================================================

diff -r d21671811dff -r 942a92874526 js/xpconnect/src/xpcprivate.h
--- a/js/xpconnect/src/xpcprivate.h	Tue Feb 18 11:14:49 2020 +0000
+++ b/js/xpconnect/src/xpcprivate.h	Sat Feb 22 04:56:52 2020 +0000
@@ -2237,6 +2237,7 @@
   bool CSSRule : 1;
   bool Directory : 1;
   bool Document : 1;
+  bool DOMException : 1;
   bool DOMParser : 1;
   bool DOMTokenList : 1;
   bool Element : 1;